tbx/project_styleguide/templates/patterns/pages/blog/blog_listing.html

```diff
@@ -14,9 +14,8 @@
 
         <ul class="page-listing page-listing--blog">
             {% for post in blog_posts %}
-                {% firstof post.url post.link_external as link %}
-
-                {% include "patterns/molecules/listing/listing--avatar.html" with link=link avatar=post.author.image title=post.title name=post.author.name job_title=post.author.role date=post.date reading_time=post.read_time tags=post.tags tag_link_base=blog_index_url base_class="page-listing__item" %}
+                {% pageurl post as link %}
+                {% include "patterns/molecules/listing/listing--avatar.html" with link=link avatar=post.first_author.image title=post.title name=post.first_author.name job_title=post.first_author.role date=post.date reading_time=post.read_time tags=post.tags tag_link_base=blog_index_url base_class="page-listing__item" %}
```

Comment:
the author fields need updating in the pattern library here

Main branch blog listing in pattern library:

![Screenshot 2024-04-30 at 11 59 19](https://github.com/torchbox/torchbox.com/assets/771869/4e96f96e-9503-4eec-88af-d9b69c22f937)
 
This branch:

![Screenshot 2024-04-30 at 12 01 06](https://github.com/torchbox/torchbox.com/assets/771869/784f6f80-4161-4315-9116-c61c01938894)